### ISSUE TITLE
Unable to connect to WebDAV

### ISSUE THREAD

**Issue by Red-Ex** [sodenn/2do-txt#761]
Hello @sodenn,

thank you for the great Work:)!
Do I am something wrong by connecting the App to a WebDAV-Server?
I tried https and http and get always a connection error.
Is there an option to look into the logs and find a reason or is there an example for a serverconfiguration and an connection that I can try?

Thank you!

**Comment by sodenn** [comment]
Hi @Red-Ex. Is the problem in the desktop app or the iOS app? Which version are you currently using?

**Comment by Red-Ex** [comment]
Hi @sodenn.
The Problem occurs on both platforms. I think it is relying on the webdav-backend. I've first tried the WebDAV-Role on a Windows IIS-Server. After that fails I tried a Docker container from [bytemark/webdav](https://hub.docker.com/r/bytemark/webdav/). Unfortunately that fails too. As the last option I tried a nextcloud-Instance which was successfully. So currently I think, that there are differences between that implementation or you have build the sync especialy for nextcloud. As it works with nextcloud, I will use nextcloud:). I'm planning a nextcloud-instance in my HomeLab, so that is the best reason, cause the sync is really smooth an makes the app very valuable for me:). 
Thank you for the effort:)!
